content/docs/1_guide/4_content/4_fields/guide.txt

```diff
@@ -50,7 +50,7 @@ In general, you are free to name your fields like you want to, but there are two
 Field content is accessible in your templates via the field's name:
 
 ```php
-<?= $page->title()->html() ?>
+<?= $page->title()->escape() ?>
 <?= $page->text()->kirbytext() ?>
```

Comment:
What about `kirbytext()` and `toDate()` here?

Reply:
I fixed the `kirbytext()` example. `toDate()` is fairly safe with a fixed format string.